src/Capgemini.PowerApps.PackageDeployerTemplate/Config/EmailRouterAccessApproval.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Capgemini.PowerApps.PackageDeployerTemplate.Config
+{
+    /// <summary>
+    /// EmailRouter Access Approval.
+    /// </summary>
+    public enum EmailRouterAccessApproval
```

Comment:
The config folder is the Import Config settings. These Dataverse values would be better aligned in the Constants class such as `Constants.Workflow.StateCodeActive`.